### PR TITLE
standarized on one delay(ms) call

### PR DESCRIPTION
**Pull request by craiglink**
The code base currently has 3 calls to delay for X number of milliseconds.

This change standarized on the default delay(ms) call which just calls vTaskDelay() under the covers.  It also updates the x86 support to do the same.  

The custom delay_ms(ms) in the code base has the same implementation as delay(ms), so those calls have been updated to just use delay(ms).

### CODE REVIEW

File: FluidNC/src/Motors/Dynamixel2.cpp [hunk 1]
```diff
@@ -73,7 +73,7 @@ namespace MotorDrivers {
 
         // servos will blink in axis order for reference
         LED_on(true);
-        vTaskDelay(100);
```
Comment (craiglink) [inline FluidNC/src/Motors/Dynamixel2.cpp:76]: this was probably a previous minor bug that should have been `vTaskDelay(100 / portTICK_PERIOD_MS);`

File: FluidNC/src/Motors/RcServo.cpp [hunk 2]
```diff
@@ -92,8 +90,8 @@ namespace MotorDrivers {
             float home_time_sec = (axisConfig->_maxTravel / axisConfig->_maxRate * 60 * 1.1);  // 1.1 fudge factor for accell time.
 
             _disabled = false;
-            set_location();                    // force the PWM to update now
-            vTaskDelay(home_time_sec * 1000);  // give time to move
```
Comment (craiglink) [inline FluidNC/src/Motors/RcServo.cpp:96]: this was probably a previous minor bug that should have been `vTaskDelay(home_time_sec * 1000 / portTICK_PERIOD_MS);`

File: FluidNC/src/WebUI/WebServer.cpp [hunk 3]
```diff
@@ -860,7 +860,7 @@ namespace WebUI {
                     //Upload write
                     //**************
                 } else if (upload.status == UPLOAD_FILE_WRITE) {
-                    vTaskDelay(1 / portTICK_RATE_MS);
```
Comment (craiglink) [inline FluidNC/src/WebUI/WebServer.cpp:863]: this previously would evaluation to 1 or 0 depending on the hardware's clock rate.  Was that intentional?

File: FluidNC/src/Protocol.cpp [hunk 4]
```diff
@@ -124,10 +124,10 @@ char activeLine[Channel::maxLine];
 bool pollingPaused = false;
 void polling_loop(void* unused) {
     // Poll the input sources waiting for a complete line to arrive
-    for (; true; /*feedLoopWDT(), */ vTaskDelay(0)) {
+    for (; true; /*feedLoopWDT(), */ delay(0)) {
         // Polling is paused when xmodem is using a channel for binary upload
         if (pollingPaused) {
-            vTaskDelay(100);
```
Comment (craiglink) [inline FluidNC/src/Protocol.cpp:130]: this was probably a previous minor bug that should have been `vTaskDelay(100 / portTICK_PERIOD_MS);`

**Review by MitchBradley — COMMENTED** [review]
This is a good idea but did you see my discord post where I analyzed all the delays and suggested a different approach?  I would rather use an internally-defined delay function instead of going directly to the Arduino framework's delay(), for future portability in case we are finally able to ditch Arduino.

**Comment by craiglink** [comment]
I was debating that as well. I chose to go with the Arduino one because of the number of 3rd party Arduino based libraries included in the project. But I’d be fine other other way as well. Want me to do that with this PR?

**Comment by craiglink** [comment]
The only discord comments I saw from you are the ones I responded too in discord.

**Comment by MitchBradley** [comment]
https://discord.com/channels/780079161460916227/1258511678287646730/1258511678287646730

If you could implement that plan, it would be great.

**Comment by craiglink** [comment]
looks like you beat me to it here - https://github.com/bdring/FluidNC/pull/1255/files

**Pull request closed without merging**